tests/test_multiplane.py

```diff
@@ -50,7 +50,9 @@ def test():
         )
 
     # Use same cosmology
-    cosmo_ap = FlatLambdaCDM_ap(cosmology.h0.value, cosmology.Om0.value, Tcmb0=0)
+    cosmo_ap = FlatLambdaCDM_ap(
+        cosmology.h0.cpu().value, cosmology.Om0.cpu().value, Tcmb0=0
```

Comment:
looks like you just need to switch it to `.value.cpu()` so the tensor is created first, then moved to cpu